packages/bits/spec/components/menu/menu.atom.ts

```diff
@@ -13,6 +14,9 @@ import { MenuItemAtom } from "./menu-item.atom";
 
 export class MenuAtom extends Atom {
     public static CSS_CLASS = "nui-menu";
+
+    public static getAllMenuItems = () => element.all(by.css(".nui-menu-item:not(.nui-menu-item--header"));
```

Comment:
I'm wondering if it might be more user-friendly to return an array of MenuItemAtom's instead of an ElementFinderArray.

Reply:
The broken method does exactly that and fails to deliver. So, until we know the root cause of the problem, I played safe and worked with ElementFinderArray until we fix the broken method which delivers menu item atoms.

Reply:
okay